build/eslint/xfail.list

```diff
@@ -1,298 +1,206 @@
+apps/communications/contacts/test/unit/import/import_test.js:348:13: Using postMessage() with wildcard targets is not allowed. [Error/no-wildcard-postmessage/no-wildcard-postmessage]
+apps/communications/contacts/test/unit/import/import_test.js:482:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/link_test.js:77:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/mock_multiple_select_dom.js:10:3: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/navigation_test.js:18:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/search_test.js:52:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/utilities/tagSelector_test.js:35:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/views/details_test.js:170:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/views/form_test.js:88:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/views/ice_settings_test.js:107:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/views/ice_test.js:63:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/views/ice_test.js:76:7: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/views/multiple_select_test.js:22:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/views/settings_test.js:134:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/communications/contacts/test/unit/views/settings_test.js:474:7: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/costcontrol/js/settings/settings.js:170:7: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/costcontrol/test/unit/formatting_test.js:46:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/costcontrol/test/unit/formatting_test.js:228:7: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/email/build/email.build.js:3:10: Parsing error: Unexpected token [Error]
+apps/email/build/gelam_worker.build.js:3:10: Parsing error: Unexpected token [Error]
+apps/email/build/main-frame-setup.build.js:3:6: Parsing error: Unexpected token [Error]
```

Comment:
I'm surprised this eslint policy change introduces new innerHTML violation. Why?

Most of the files in email/build/ are actually json, not js. Maybe we should file a follow-up bug to rename them to json?